src/pages/partners/payment-agent/_faq-data.tsx

```diff
@@ -8,50 +8,20 @@ const General = () => {
     const { is_p2p_allowed_country } = React.useContext(DerivStore)
     return (
         <>
-            <Header
-                as="p"
-                type="paragraph-1"
-                mt="16px"
-                tabletL={{
-                    type: 'paragraph-2',
-                }}
-            >
+            <Header as="p" type="paragraph-1" mt="16px">
                 {localize('What is the Deriv Payment Agent Programme?')}
             </Header>
-            <Header
-                as="p"
-                type="paragraph-1"
-                mt="8px"
-                weight="normal"
-                tabletL={{
-                    type: 'paragraph-2',
-                }}
-            >
+            <Header as="p" type="paragraph-1" mt="8px" weight="normal">
```

Comment:
```suggestion
            <StyledHeader as="p" mt="8px" weight="normal">
```

Same for the rest of the Headers